run/pubsub/main.py

```diff
@@ -14,15 +14,25 @@
 
 # [START cloudrun_pubsub_server_setup]
 # [START run_pubsub_server_setup]
+# [START cloudrun_pubsub_server]
 import base64
 import os
 
 from flask import Flask, request
 
 
 app = Flask(__name__)
+
+if __name__ == "__main__":
+    PORT = int(os.getenv("PORT")) if os.getenv("PORT") else 8080
+
+    # This is used when running locally. Gunicorn is used to run the
+    # application on Cloud Run. See entrypoint in Dockerfile.
+    app.run(host="127.0.0.1", port=PORT, debug=True)
```

Comment:
Just confirming you want this to be `debug=True`?

Reply:
updated in last commit